Exploration Queries/InputEntity_Host/ProcessBlockedNonMS.yaml

```diff
@@ -0,0 +1,25 @@
+Id: f87b2afb-068f-4734-88a0-94560309f9c7
+DisplayName: Processes blocked from loading non-Microsoft-signed binaries 
+Description: Exploit protection blocked the launch of a process from an image file that is not signed by Microsoft
+InputEntityType: Host
+InputFields:
+  - HostName
+OutputEntityTypes:
+  - Process
+QueryPeriodBefore: 1d
+QueryPeriodAfter: 1d
+DataSources:
+  - DeviceEvents
+Tactics:
+  - Persistence
+  - LateralMovement
+query: |
+  let BlockedUnsigned = (v_Host_HostName:string){
+  DeviceEvents
+  | where ActionType == "ExploitGuardNonMicrosoftSignedBlocked" and FileName !hassuffix ".ni.dll"
+  | where v_Host_HostName =~ tostring(split(DeviceName, '.')[0])
+  | summarize Count=count() by Process_ProcessId=InitiatingProcessId, Process_CommandLine=InitiatingProcessCommandLine, Process_CreationTimeUtc=InitiatingProcessCreationTime, Process_Host_UnstructuredName=DeviceName
```

Comment:
DeviceEvents does not have a column named InitiatingProcessCreationTime

Reply:
Wow! Glad you caught that. Removing this for now and taking a task to understand why this is missing

Reply:
This coulmn is missing on the connector side. This will be fixed soon, and I will reintroduce the field mapping